doc/api/process.md

```diff
@@ -373,7 +373,10 @@ It is important to take note of the following:
 * `SIGTERM` is not supported on Windows, it can be listened on.
 * `SIGINT` from the terminal is supported on all platforms, and can usually be
   generated with `CTRL+C` (though this may be configurable). It is not generated
-  when terminal raw mode is enabled.
+  when terminal raw mode is enabled. If your program listens for `SIGINT`, it is
```

Comment:
Sorry, I am -1 on this text.

Did you not notice that this exact behaviour is documented a dozen lines above? And that you added this for SIGINT, but not SIGTERM? Not to mention that it is absolutely not the program's responsibility to call process.exit(). There is no requirement to exit after handling SIGINT (or SIGTERM, or any other signal that you can listen on).